### PR TITLE
Add support for Django 3.1

### PR DESCRIPTION
**Pull request by Djailla**

### CODE REVIEW

**Comment by FinalAngel** [comment]
👍

File: README.rst [hunk 1]
```diff
@@ -62,7 +62,7 @@ Django to perform an ``INNER JOIN`` to fetch the model fields from the database.
 While taking this in mind, there are valid reasons for using subclassed models.
 That's what this library is designed for!
 
-The current release of *django-polymorphic* supports Django 1.11, 2.0, 2.1, 2.2 and Python 2.7 and 3.5+ is supported.
+The current release of *django-polymorphic* supports Django 2.1, 2.2, 3.0 and Python 3.5+ is supported.
```
Comment (yakky) [inline README.rst:65]: 3.1 is missing here :)

**Comment by peterfarrell** [comment]
@yakky Yay! Would it be possible to push a release to PyPI?

**Comment by yakky** [comment]
@peterfarrell we need @chrisglass or @vdboor for this

**Comment by chrisglass** [comment]
I can look into doing a release with this branch, although I haven't done anything like that in a while so it might be a bit longer than otherwise :)

**Comment by chrisglass** [comment]
This branch, plus fixing a bit of a test seems to manage to make tox happy on my machine for at least more than 0 cases :)

So I'll merge this now, then send a quick commit to fix the cases that succeed on my machine (py38-django21 and py38-django22).

I would appreciate it if someone with more up-to-date knowledge of django3* could fix the py38-django30 and py38-django31 tests however (in another PR).

**Pull request merged**